app/src/main/res/layout/user_map_activity.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Cirrus says we can use merge instead of FrameLayout, maybe we could check that ?